src/test/java/com/flightstats/hub/webhook/WebhookTest.java

```diff
@@ -141,4 +147,12 @@ public void testIsTagPrototype() throws Exception {
         assertTrue(twh.isTagPrototype());
     }
 
+    @Test
+    public void testSecondaryMetricsReporting() {
+        Webhook withDefaults = this.webhook.withDefaults();
+        assertFalse(withDefaults.isSecondaryMetricsReporting());
+        String json = "{ \"secondaryMetricsReporting\": \"true\" }";
```

Comment:
is true a string or boolean? if boolean, quotes not required.

Reply:
true :) pun intended the quotes are not necessary